source/fileformats.md

```diff
@@ -0,0 +1,20 @@
+# File Formats
+
+The Particle Accelerator Lattice Standard (PALS) documents objects and their properties.
+For readability, this standard uses YAML-style examples, but the PALS schema can be implemented in a variety of file formats.
```

Comment:
Should reference the `Syntax Used in this Document` section in the introduction.md file.